dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -451,3 +451,16 @@ contextPoolConfig=Context Pool Properties
 contextPoolConfig.desc=Properties of the context pool.
 
 contextPoolConfig$Ref=Context pool properties reference
+
+bindAuthMechanism=Bind Authentication Mechanism
+bindAuthMechanism.desc=The authentication mechanism for binding to the LDAP server when searching or modifying an LDAP entry.
+
+bindAuthMechanism.none=Anonymous bind to the directory service.
+bindAuthMechanism.simple=Use the bindDN and bindPassword to bind to the directory service. The bindDN and bindPassword attributes are required.
+bindAuthMechanism.GSSAPI=GSS-API Kerberos v5 (krb5) support to bind to the directory service. The krb5Principal attribute is required
+
+krb5Principal=Kerberos Principal Name
+krb5Principal.desc=Kerberos principal name or Kerberos service principal name that is used to authenticate with the Key Distribution Center (KDC) when using the GSSAPI bindAuthMechanism.
+
+krb5TicketCache=Kerberos Ticket Cache File
```

Comment:
We have these defined in data source, should we keep them the same?
krb5Principal=Kerberos principal name
krb5Principal.desc=The name of the Kerberos principal name or Kerberos service name to be used.

krb5TicketCache=Kerberos ticket cache
krb5TicketCache.desc=The file location where Kerberos credentials for the Kerberos principal name or service name will be stored. Also known as the Kerberos credential cache (ccache)

Reply:
I did use the krb5TichetCache desc, but I changed the tense. The krb5Princpal is from tWAS and I added `when using the GSSAPI bindAuthMechanism` onto it. Do you want to swap it back to the simpler version from datasource?

Reply:
I would keep it the same with data source for Liberty.

Reply:
Will swap to match datasource